src/main/java/com/coveros/selenified/element/check/Contains.java

```diff
@@ -292,20 +292,20 @@ default String checkValue(String expectedValue, double waitFor, double timeTook)
      * @param timeTook     - the amount of time it took for wait for something (assuming we had to wait)
      * @return Integer: the number of select options of the element. -1 will be returned if the element isn't present or a select
      */
-    default int checkSelectOptions(int numOfOptions, double waitFor, double timeTook) {
+    int checkSelectOptions(int numOfOptions, double waitFor, double timeTook) {
         // record the action, and check for select
-        if (!isPresentSelect(getElement().prettyOutput() + " with number of select values equal to <b>" + numOfOptions + "</b>", waitFor)) {
+        if (!isPresentSelect(this.element.prettyOutput() + " with number of select values equal to <b>" + numOfOptions + "</b>", waitFor)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " with number of select values equal to <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)